docs/source/tutorial-tracking-ui.rst

```diff
@@ -0,0 +1,140 @@
+.. _tutorial-tracking-ui:
+
+Compare and choose with the MLflow Tracking UI
+==============================================
+
+As a data scientist or MLOps professional, you need to be able to quickly filter and review a large number of runs to find the best model for your use case. The MLflow Tracking UI is a web application that visualizes the results of experiments and runs for which you used the MLflow Tracking API.
+
+.. important:: 
+
+   You should complete the :ref:`quickstart`, or even better, the :ref:`tutorial-tracking` before starting this how-to. This document assumes that you have several runs for which you've used the MLflow Tracking API to log parameters, metrics, and artifacts.
+
+Running the Tracking UI
+-----------------------
+
+If you have an instance of the MLflow Tracking Server running, you can access the Tracking UI by browsing to its location (for instance, ``http://localhost:5000``).
+
+You can also run the Tracking UI from the command-line with the ``mlflow ui`` command. It accepts many of the same arguments as the Tracking Server (``--port``, ``--host``, ``--backend-store-uri``, etc.). By default, ``mlflow ui`` will look for the ``MLFLOW_TRACKING_URI`` environment variable and use that as the backend store URI. If you do not set this environment variable and do not use the ``--backend-store-uri`` parameter, the Tracking UI will visualize the **mlruns/** subdirectory of the current working directory.
+
```

Comment:
Can we add an example here of setting some directory on the local FS as the `backend-store-uri` that houses the run data that we've recorded and the artifact data stored in `artifacts-destination`, then show the actual cli command to start the UI pointing to that location, also showing a localhost with the correct port opened for the UI server (just to make it a copyable instruction for users)?

Reply:
So two instances, the server on :5000 (say) and the ui on :5001 (say) but instead of both using `MLFLOW_TRACKING_URI`, both doing it with `--backend-store-uri` and `--artifacts-destination` args?

Reply:
that would be great :)